examples/robot/lifelong_learning_bench/README.md

```diff
@@ -0,0 +1,108 @@
+
+## Step 1. Ianvs Preparation
+
+First, we download the code of Ianvs. Assuming that we are using `/ianvs` as workspace, Ianvs can be cloned with `Git`
+as:
+
+``` shell
+mkdir /ianvs
+cd /ianvs #One might use another path preferred
+
+mkdir project
+cd project
+git clone https://github.com/kubeedge/ianvs.git   
+```
+
+
+Then, we install third-party dependencies for ianvs. 
+``` shell
+sudo apt-get update
+sudo apt-get install libgl1-mesa-glx -y
+python -m pip install --upgrade pip
+
+cd ianvs 
+python -m pip install ./examples/resources/third_party/*
+python -m pip install -r requirements.txt
+```
+
+We are now ready to install Ianvs. 
+``` shell
+python setup.py install  
+```
+
+## Step 2. Dataset Preparation
+
+Datasets and models can be large. To avoid over-size projects in the Github repository of Ianvs, the Ianvs code base does
+not include origin datasets. Then developers do not need to download non-necessary datasets for a quick start.
+
+``` shell
+mkdir /data
+cd /data
+mkdir datasets
+cd datasets
+python -m pip install kaggle
+kaggle datasets download -d hsj576/cloud-robotics
+unzip cloud-robotics.zip
+```
+
+The URL address of this dataset then should be filled in the configuration file ``testenv.yaml``. In this quick start,
+we have done that for you and the interested readers can refer to [testenv.yaml](https://ianvs.readthedocs.io/en/latest/guides/how-to-test-algorithms.html#step-1-test-environment-preparation) for more details.
+
+<!-- Please put the downloaded dataset on the above dataset path, e.g., `/ianvs/dataset`. One can transfer the dataset to the path, e.g., on a remote Linux system using [XFTP].  -->
+
+
+Related algorithm is also ready in this quick start. 
+``` shell
+export PYTHONPATH=$PYTHONPATH:/ianvs/project/ianvs/examples/robot/lifelong_learning_bench/testalgorithms/rfnet/RFNet
+```
+
+The URL address of this algorithm then should be filled in the configuration file ``algorithm.yaml``. In this quick
+start, we have done that for you and the interested readers can refer to [algorithm.yaml](https://ianvs.readthedocs.io/en/latest/guides/how-to-test-algorithms.html#step-1-test-environment-preparation) for more details.
+
+## Step 3. Ianvs Execution and Presentation
+
+We are now ready to run the ianvs for benchmarking. 
+
+``` shell
+cd /ianvs/project/ianvs
+ianvs -f examples/robot/lifelong_learning_bench/benchmarkingjob.yaml
+```
+
+Finally, the user can check the result of benchmarking on the console and also in the output path(
+e.g. `/ianvs/lifelong_learning_bench/workspace`) defined in the benchmarking config file (
+e.g. `benchmarkingjob.yaml`). In this quick start, we have done all configurations for you and the interested readers
+can refer to [benchmarkingJob.yaml](https://ianvs.readthedocs.io/en/latest/guides/how-to-test-algorithms.html#step-1-test-environment-preparation) for more details.
+
+The final output might look like this:   
```

Comment:
Is this the real output of this example? Could you show us some systematic performance, such as latency of the whole pipeline?

Reply:
> Is this the real output of this example? Could you show us some systematic performance, such as latency of the whole pipeline?

Yes, it is the real output of this example. The tutorial of this example could be found in [here](https://github.com/hsj576/ianvs-lifelong-learning-tutorial/blob/main/tutorial/tutorial.md).
![result](https://github.com/kubeedge/ianvs/assets/55343399/545fe557-abfd-44d6-89d1-97730095f29e)